python/pyhail/dataset.py

```diff
@@ -1040,6 +1385,29 @@ def print_schema(self, output=None, attributes=False, va=False, sa=False, print_
     def rename_samples(self, input):
         """Rename samples.
 
+        **Example**
+
+        >>> vds = (hc.read('example.vds')
+        >>>  .rename_samples('sample.map'))
+
+        **Notes**
+
+        The input file is a two column, tab-separated file.  The first column is the current sample
+        name, the second column is the new sample name.  Samples which do not
+        appear in the first column will not be renamed.  Lines in the input that
+        do not correspond to any sample in the current dataset will be ignored.
+
+        ``exportsamples`` can be used to generate a template for renaming
```

Comment:
refer to exportsamples as:

:py:meth:\`~pyhail.VariantDataset.export_samples\`